src/app/organizer/Columns.tsx

```diff
@@ -160,7 +160,10 @@ export function getColumns(
             return <ItemStatValue stat={stat} item={item} />;
           },
           defaultSort: statInfo.lowerBetter ? SortDirection.ASC : SortDirection.DESC,
-          filter: (value) => `stat:${_.invert(statHashByName)[statHash]}:>=${value}`,
+          filter: (value) =>
+            `stat:${_.invert(statHashByName)[statHash]}:${
+              _.invert(statHashByName)[statHash] === `rof` ? `` : `>`
+            }=${value}`,
```

Comment:
```suggestion
          filter: (value) => {
            const statName = _.invert(statHashByName)[statHash];
            return `stat:${statName}:${statName === 'rof' ? '=' : '>='}${value}`;
          },
```

Reply:
Great fix for the readability. My bad on not thinking of using the variable to abstract it out earlier. Thank you!